styles/widgets/material/size-schemes/default.less

```diff
@@ -35,8 +36,8 @@
 
 @PIVOTGRID_EMPTY_COLUMN_WIDTH: 19px;
 
-@PIVOTGRID_COMMON_LEFT_RIGHT_PADDING: 10px;
-@PIVOTGRID_COMMON_TOP_BOTTOM_PADDING: 7px;
+@PIVOTGRID_COMMON_LEFT_RIGHT_PADDING: 14px;
```

Comment:
Looks like it's wrong... I found 10 places where these two constants are used. Is there really the reason to change values in all 10 places?

Reply:
You are right. I'll change this.